README.md

```diff
@@ -18,35 +18,50 @@ these checks verify that the plugin is available for users of napari
 to install, and would register entry points with napari. 
 
 There are two parts to the validation tool. One part is a 
-command line interface and the other is a pytest fixture.
+command line interface, and the other is a pytest fixture.
 
 ### Command Line Interface (CLI) Usage
 The CLI can be used by continuous integration (CI) pipelines to perform a 
-quick verification of a plugins setup without any specific input required. 
-It serves as a quick "sanity check". (It is also accessible from python in validation.py)
+quick verification of a plugin setup without any specific input required. 
+It serves as a quick "sanity check". (It is also accessible via Python in validation.py)
 
 The tool can be run as `npd <cmd>`. where currently we support cmd `validate`:
 
-`npd validate`: validate classifiers and function hooks can be recognized by napari.
-The validation would run on built packages under dist folder to check if they are annotated
-properly with framework classifier, and validate hooks are properly annotated that they can be found
-by napari.
+Before running `npd validate`, make sure you have prepared the repo for validation:
```

Comment:
Can we also offer the option to scan my current directory or be able to specify a directory instead of my current python environment so that I can validate my plugin as I'm developing it and before I publish to Pypi?